_docs_integrate/integration-example.md

```diff
@@ -222,23 +222,22 @@ To do so, we [synchronize updates of the Backbone]({% link _docs_use-cases/use-c
 
 {% include rapidoc api_route_regex="^post /api/v2/Account/Sync$" %}
 
-In the Response we will receive the Relationship.
+We can now [get the Relationship]({% link _docs_use-cases/use-case-transport-query-relationships.md %}).
+
+{% include rapidoc api_route_regex="^get /api/v2/Relationships$" %}
```

Comment:
maybe you should explain

1. how to filter for the new relationship for example by the template
2. also say that you should listen for the event RelationshipChanged

Reply:
b/c we don't want the people to query here, instead to listen to events

Reply:
We already have a disclaimer at the top to disable the Sync Module, I'll expand it. The Relationships and Messages tutorials will need some event adaptation, the requests tutorial only needs to be updated with the sse-Module.
I would like to do the eventing adaptions in a different PR though